### DISCUSSION TITLE
The music for my new week isnt working

### DISCUSSION THREAD

**Discussion by ElbryanWyndon** [FunkinCrew/Funkin#1551] [Q&A]
I don't know what to do, here are some images of my music file setup, if its right then I don't know whats wrong...
![image](https://user-images.githubusercontent.com/83390237/118075717-71f6ca80-b365-11eb-81f3-88ca0582a226.png)
![image](https://user-images.githubusercontent.com/83390237/118075724-76bb7e80-b365-11eb-9328-019b2bc22c03.png)

**Comment by ElbryanWyndon** [comment]
The game runs but whenever I try to launch any of the custom songs it doesn't work.

**Comment by steviegt6** [comment]
Please stop making issues on the Funkin issue tracker for this kind of crap.

**Comment by ElbryanWyndon** [comment]
well where else do i do it

**Comment by ElbryanWyndon** [comment]
i have no experience coding, this is the only place i know

**Comment by ghost** [comment]
i think lowercase the folder name like

song
vs
Song

but idk

**Comment by ElbryanWyndon** [comment]
tried that, didnt work

**Comment by R-code-0** [comment]
> tried that, didnt work

Where is your song data?

**Comment by ghost** [comment]
> tried that, didnt work
> 
> 

have you put in your charts in `\assets\data\song`?

**Comment by Windows1Door2Daniel** [comment]
in debug mode put the song slot as inst

**Comment by Windows1Door2Daniel** [comment]
i think the folder should be music

every song file are in the music folder witout being separated

you should put 2 files with the same name (inst and voices at the end) [example: Rickroll_inst.ogg and Rickroll_voices.ogg

in data folder create a folder with the JSON files with the debug mode in game (is better do 3 JSONs for each dificulty)

you can test in debug mode by presing 7 in any week in the game writing the song name spot the name of your song click reload JSON (If it show a error mesage that means it dont have JSON file the song) (Reload Normal JSON i think) then press [Enter]

**Comment by Windows1Door2Daniel** [comment]
you need HaxeFlixel

**Comment by ElbryanWyndon** [comment]
I have haxeflixel, ive worked out that issue but i have a much bigger isue now
